projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -9,37 +9,37 @@ import {
   Output,
   PLATFORM_ID,
   ViewContainerRef
-} from "@angular/core";
-import { AfterViewInit, OnChanges, OnDestroy, OnInit } from "@angular/core";
-import tippy from "tippy.js";
-import { NgChanges, TIPPY_CONFIG, TIPPY_REF, TippyConfig, TippyInstance, TippyProps } from "./tippy.types";
-import { dimensionsChanges, inView, onlyTippyProps, overflowChanges } from "./utils";
-import { fromEvent, merge, Subject } from "rxjs";
-import { switchMap, takeUntil } from "rxjs/operators";
-import { isComponent, isString, isTemplateRef, ViewService } from "@ngneat/overview";
-import { ViewOptions, ViewRef } from "@ngneat/overview";
-import { Content } from "@ngneat/overview";
-import { isPlatformServer } from "@angular/common";
+} from '@angular/core';
+import { AfterViewInit, OnChanges, OnDestroy, OnInit } from '@angular/core';
```

Comment:
I am not sure whether you should change the "code formatting"

Reply:
it was changed here:
https://github.com/ngneat/helipopper/pull/40/files#diff-e42472dd65561fe56344281078077fb924cddca92821eadfbc2318cca3f89c9aR2

the code is formatted anyway when creating a commit